### PR TITLE
Highlight clock when out of sync

### PR DESCRIPTION
**Pull request by mnishiguchi**
resolves https://github.com/nerves-project/nerves_motd/issues/74

CleanShot 2022-10-30 at 19 19 16@2x

TODO

- [x] resolve merge conflict
- [x] add (unsynchronized) 
- [x] yellow text instead of yellow background

### CODE REVIEW

**Review by jjcarstens — COMMENTED** [review]
I have a couple thoughts here:
* Is changing color enough of a context? I'm not sure how easy it is to deduce that yellow == unsynchronized. Maybe we need to include `(unsynchronized)` next to it as well?
* I'm going back and forth on having yellow background vs yellow text. It seems like yellow text might be less aggressive?

File: test/nerves_motd_test.exs [hunk 1]
```diff
@@ -113,6 +113,15 @@ defmodule NervesMOTDTest do
     assert capture_motd() =~ ~r/Clock        : \d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2} \w{3}/
   end
 
+  test "Clock gets hightlighed when not synchronized" do
```
Comment (fhunleth) [inline test/nerves_motd_test.exs:116]:
```suggestion
  test "Clock gets highlighted when not synchronized" do
```

**Review by fhunleth — APPROVED** [review]
Looks good.

**Comment by fhunleth** [comment]
I missed that CI is currently broke. It will look good after that's fixed. Also, could you update CI to test with Elixir 1.14.3? Elixir 1.9 and 1.10 and be removed, since other packages require at least 1.11 anyway.

**Comment by mnishiguchi** [comment]
✅ quick test on a mangopi_mq_pro as of [8318380](https://github.com/nerves-project/nerves_motd/pull/97/commits/83183803ba5338f6701766d865431a55f84eb43d)

- for a few minutes after the boot, the clock is out of synch and then it gets synched

**Comment by mnishiguchi** [comment]
> I missed that CI is currently broke. It will look good after that's fixed. Also, could you update CI to test with Elixir 1.14.3? Elixir 1.9 and 1.10 and be removed, since other packages require at least 1.11 anyway.

I'll update the CI in a separate PR.

**Pull request merged**